src/coreclr/jit/CMakeLists.txt

```diff
@@ -692,6 +692,9 @@ endif (CLR_CMAKE_TARGET_ARCH_RISCV64)
 # Note as well that we need this "CLR_CMAKE_BUILD_LLVM_JIT", defined by the build scripts,
 # because there is no way in CMake to ask "what --target's' am I being asked to configure?".
 if (CLR_CMAKE_BUILD_LLVM_JIT)
+    # LLVM headers require C++17.
+    set(CMAKE_CXX_STANDARD 17)
```

Comment:
Can this be an LLVM-Jit-target-specific property instead: `set_target_properties(${TARGETDETAILS_TARGET} PROPERTIES CXX_STANDARD 17)`?

Reply:
Yes, moved it around a bit.